convert2rhel/unit_tests/logger_test.py

```diff
@@ -85,7 +85,7 @@ def test_log_format(self):
             logging.DEBUG, tempstr)
         self.check_formatter_result(
             logging.INFO, self.test_msg)
-        tempstr = "WARNING - %s" % self.test_msg
+        tempstr = "\033[93mWARNING - %s\033[0m" % self.test_msg
         self.check_formatter_result(
             logging.WARNING, tempstr)
```

Comment:
Add a test for when removing/escaping coloring
```suggestion
        tempstr = "\033[93mWARNING - %s\033[0m" % self.test_msg
        self.check_formatter_result(
            logging.WARNING, tempstr)

        custom_formatter.escape_colors(True)
        tempstr = "WARNING - %s" % self.test_msg
        self.check_formatter_result(
            logging.WARNING, tempstr)
```

Reply:
I believe default = True it's better, since we have colors in all info (info, warning and error)... and we don't have colors only in file.

Reply:
It doesn't really matter, but due to the choice of words with `escape_colors` rather than something like `enable_colors` it should be `False` by default. Likewise, since CLI is our main use case, we should go by that rather than by logs - i.e. colors enabled by default.

Also I just realized, but I believe that some orchestration and logging tools will listen to stdout rather than logs too. It might make sense to have a CLI option to disable it as well to not mess up their own logging

Reply:
Just missing the test for when disabling colors like in the suggestion here